src/api/helpers/get-expiring-applications/index.test.ts

```diff
@@ -0,0 +1,97 @@
+import getExpiringApplications from '.';
+import { createMultipleFullApplications, getKeystoneContext } from '../../test-helpers';
+import applications from '../../test-helpers/applications';
+import { Application, Context } from '../../types';
+import { dateInTheFutureByDays } from '../date';
+import { APPLICATION } from '../../constants';
+
+const { IN_PROGRESS, ABANDONED, SUBMITTED } = APPLICATION.STATUS;
+const { REMINDER_DAYS } = APPLICATION.SUBMISSION_DEADLINE_EMAIL;
+
+const assertApplicationValues = (application: Application, expectedApplication: Application) => {
```

Comment:
Not a blocker, `expectedApplication` could be `expectedApplicationId` string to simplify line 12